chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,848 @@
+  loadSettings(popup) {
+    const self = this;
+    const setID = self.CHROME_SETTINGS_ID;
+    chrome.storage.sync.get(setID, function (response) {
+      if (response?.[setID]) {
+        // NOTE (Ace, 7-Feb-2017): Might need to store these off the app object:
+        try {
+          self.popupView.data.settings = JSON.parse(response[setID]);
+        } catch (err) {
+          g2t_log(
+            'loadSettings: JSON parse failed! Error: ' + JSON.stringify(err)
+          );
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)